public/sitemap.txt

```diff
@@ -1,11 +1,9 @@
 https://app.shapeshift.com
-https://app.shapeshift.com/#/connect-wallet
-https://app.shapeshift.com/#/demo/fox
-https://app.shapeshift.com/#/demo/defi
-https://app.shapeshift.com/#/demo/defi/earn
-https://app.shapeshift.com/#/demo/trade
-https://app.shapeshift.com/#/demo/assets
-https://app.shapeshift.com/#/demo/assets/bip122:000000000019d6689c085ae165831e93/slip44:0
-https://app.shapeshift.com/#/demo/assets/eip155:1/slip44:60
-https://app.shapeshift.com/#/demo/assets/eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d
-https://app.shapeshift.com/#/demo/assets/eip155:1/erc20:0xdc49108ce5c57bc3408c3a5e95f3d864ec386ed3
```

Comment:
- Removes demo prefix since demo wallet is now gone
- Remove /defi which is now gone
- Adds rFOX
- Adds buy-crypto
- Does *not* add lending/pools since those are non-actionable
- Does *not* add markets, not sure how crawlers crawl SPAs but just in case, no point spewing Portals